Lib/fontbakery/checks/googlefonts/axisregistry.py

```diff
@@ -204,6 +204,14 @@ def normalize_name(name):
             # on the GF Axis Registry for this specific variation axis tag.
             name = normalize_name(name_entry.toUnicode())
             expected_names = [normalize_name(n) for n in fallbacks.keys()]
+
+            # Allow an Italic AxisValue for the slnt axis which can be any
+            # user value.
+            if axis.AxisTag == "slnt" and name == "Italic":
```

Comment:
I think we need another elif to clear Niko's example

```PYTHON
elif axis.AxisTag == "slnt" and name not in ["Italic", "Normal/default...whatever"]:
    yield FAIL...
```

I can do this after lunch.

Reply:
@m4rc1e Ping ping. What do we actually need to be testing for here?

Reply:
We want to ensure that STAT AxisValues for the slnt are either named "Italic" or "Normal" only. We don't accept "Oblique" etc.